src/style/colors.ts

```diff
@@ -15,7 +15,11 @@ export type DesignToken = {semantic?: [string]; resource_paths?: [string]; toStr
 export type TokensOptions = {primaryColor: string};
 export type GetColorTintOptions = {avoidReverseOnDark?: boolean};
 export type GetColorByHexOptions = {validColors?: string[]};
-
+export type PaletteOptions = {
+  lightnessFix?: boolean;
```

Comment:
Maybe make this option more generic and instead of passing `lightnessFix` as boolean you can have a  `lightnessThreshold` option where you can pass the actual  threshold value you want. 
This way you easily control it from the outside without changing the source code each time

Reply:
It's not going to be as easy as passing a lightnessThreshold since the fix only applies if it meets a set range. The user will have to adjust the range and pass the value to adjust to. The boolean is about whethere I apply this range check or not. Maybe rename it adjustLightness?

Reply:
ok, let's go with `adjustLightness`